vms/platformvm/blocks/executor/backend.go

```diff
@@ -27,6 +28,7 @@ type backend struct {
 	// Note that Genesis block is a commit block so no need to update
 	// blkIDToState with it upon backend creation (Genesis is already accepted)
 	blkIDToState map[ids.ID]*blockState
+	acceptLock   sync.Locker
```

Comment:
nit: since this lock is supposed to be used in Accept only, I'd move it to acceptor, rather than keeping it in backend, where it is available to verifier and rejector too.